spec/controllers/scrapers_controller_spec.rb

```diff
@@ -0,0 +1,46 @@
+require 'rails_helper'
+
+RSpec.describe ScrapersController, type: :controller do
+
+	describe "GET #index" do
+		xit "lists all scrapers" do
+			scraper = Scraper.create
+      get :index
+      expect(assigns(:scrapers)).to eq([scrapers])
+		end
+
+		it "renders index template" do
+			get :index
+      expect(response).to render_template("index")
+		end
+	end
+
+	describe "GET #new" do
+		it "renders new template" do
+			get :new
+      expect(response).to render_template("new")
+		end
+
+		xit "renders the actual template" do
```

Comment:
This test is probably unnecessary, but if you wanted to do something like this, you need to include `render_views` in your test.